README.md

```diff
@@ -86,13 +86,24 @@ yellow_light = '#CCCC00'
 [colors.form]
 theme = 'Base16'
 
+[vcs]
+# allowed values: "git", "jj"
+backend = "git"
+
 [git]
 default_branch = 'main'
 
 [git.remote]
 enable = false
 name = 'origin'
 
+[jj]
+default_branch = "main"
+
+[jj.remote]
+enable = false
+name = "origin"
+
 [storage]
 # replace with your actual home directory
 # or any other path you'd like to use
```

Comment:
Add to the setup section the following:
```
jj git remote add origin <URL>
jj bookmark create -r @- main
jj git push --allow-new
```

These are the steps necessary to set up the remote in jj.